doc/releases/quincy.rst

```diff
@@ -0,0 +1,440 @@
+  CompleteMultipartUpload notification is sent at the end of the multipart
+  upload. The POST notification at the beginning of the upload and the PUT
+  notifications that were sent on each part are no longer sent.
+
+
+CephFS distributed file system
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* fs: A file system can be created with a specific ID ("fscid"). This is
+  useful in certain recovery scenarios (for example, when a monitor
+  database has been lost and rebuilt, and the restored file system is
+  expected to have the same ID as before).
+
+* fs: A file system can be renamed using the `fs rename` command. Any cephx
+  credentials authorized for the old file system name will need to be
+  reauthorized to the new file system name. Since the operations of the clients
+  using these re-authorized IDs may be disrupted, this command requires the
+  "--yes-i-really-mean-it" flag. Also, mirroring is expected to be disabled
+  on the file system.
+
+* MDS upgrades no longer require all standby MDS daemons to be stoped before
+  upgrading a file systems's sole active MDS.
+
+* CephFS: Failure to replay the journal by a standby-replay daemon now
+  causes the rank to be marked "damaged".
+
+Upgrading from Octopus or Pacific
+----------------------------------
+
+Quincy does not support LevelDB. Please migrate your OSDs and monitors
+to RocksDB before upgrading to Quincy.
+
+Before starting, make sure your cluster is stable and healthy (no down or
+recovering OSDs).  (This is optional, but recommended.) You can disable
+the autoscaler for all pools during the upgrade using the noautoscale flag.
+
+.. note::
+
+  You can monitor the progress of your upgrade at each stage with the
+  ``ceph versions`` command, which will tell you what ceph version(s) are
+  running for each type of daemon.
+
+Upgrading cephadm clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your cluster is deployed with cephadm (first introduced in Octopus), then
+the upgrade process is entirely automated.  To initiate the upgrade,
+
+  .. prompt:: bash #
+
+    ceph orch upgrade start --ceph-version 17.2.0
+
+The same process is used to upgrade to future minor releases.
+
+Upgrade progress can be monitored with ``ceph -s`` (which provides a simple
+progress bar) or more verbosely with
+
+  .. prompt:: bash #
+
+    ceph -W cephadm
+
+The upgrade can be paused or resumed with
+
+  .. prompt:: bash #
+
+    ceph orch upgrade pause   # to pause
+    ceph orch upgrade resume  # to resume
+
+or canceled with
+
+  .. prompt:: bash #
+
+    ceph orch upgrade stop
+
+Note that canceling the upgrade simply stops the process; there is no ability to
+downgrade back to Octopus or Pacific.
+
+
+Upgrading non-cephadm clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+   If you cluster is running Octopus (15.2.x) or later, you might choose
+   to first convert it to use cephadm so that the upgrade to Quincy
+   is automated (see above).  For more information, see
+   :ref:`cephadm-adoption`.
```

Comment:
What is this supposed to reference?

Reply:
Intended to point to https://docs.ceph.com/en/latest/cephadm/adoption/#adoption-process

Reply:
or, I guess the top of the page is better here actually (which is where I think it links) https://docs.ceph.com/en/latest/cephadm/adoption/

Reply:
Ahhhh will that actually get rendered on the website?  It just confused me because I was expecting a footer with the URL in it as I was compiling the ceph.io Release Notes.

Reply:
By the "website" do you mean something other than docs.ceph.com? If so, I'm skeptical it would work properly there and we'd probably need to include a full link.

Reply:
Nope, I meant docs.ceph.com